pre-commit.sh

```diff
@@ -8,18 +8,27 @@ echo "\n\nSaving current config\n\n"
 cp config.status tmp.status
 cp wolfssl/options.h tmp.options.h 
 
-# stash modified files not part of this commit, don't test them
-echo "\n\nStashing any modified files not part of commit\n\n"
-git stash -q --keep-index
+# stash modified files, if any, that are not part of this commit, don't test
+# them
+STASHED=0
+if ! git diff --quiet
```

Comment:
git diff --quiet returns 1 if there are modified files not added to the index, 0 otherwise.